MacWindows/MacWindows/MasterWindowController.cs

```diff
@@ -7,24 +7,27 @@
 
 namespace MacWindows
 {
-	public partial class MasterWindowController : NSWindowController
-	{
+	public partial class MasterWindowController : NSWindowController, INSWindowDelegate {
 		public MasterWindowController (IntPtr handle) : base (handle)
 		{
 		}
 
 		public override void WindowDidLoad ()
 		{
 			base.WindowDidLoad ();
+			Window.Delegate = this;
+		}
 
-			Window.DidResize += (sender, e) => {
-				// Do something as the window is being live resized
-			};
+		[Export ("windowDidResize:")]
+		public void DidResize (NSNotification notification)
+		{
+			throw new System.NotImplementedException ();
```

Comment:
These shouldn't NRE. They should be empty.

Reply:
Of course - updated! :)